omero/sysadmins/grid.rst

```diff
@@ -440,14 +440,14 @@ For example,
 
 ::
 
-    ln -s $OMERO_PREFIX/bin/omero /usr/local/bin/omero
+    ln -s $OMERODIR/bin/omero /usr/local/bin/omero
```

Comment:
Leaving this, but I assume it can be omitted. `omero` would already live in a `bin/` somewhere.